core/lib/dal/src/blocks_dal.rs

```diff
@@ -1649,6 +1763,7 @@ impl BlocksDal<'_, '_> {
             FROM
                 l1_batches
                 LEFT JOIN commitments ON commitments.l1_batch_number = l1_batches.number
+                LEFT JOIN data_availability ON data_availability.l1_batch_number = l1_batches.number
```

Comment:
What happens to existing validiums that try to 'upgrade' to run this version of the code? 
(if they have any 'batches' in flight at the time of upgrade?)

Reply:
they will be alright, the already committed batches are not affected, the non-committed will be processed through the new framework